src/main/java/lotto/domain/ProfitRateCalculator.java

```diff
@@ -0,0 +1,19 @@
+package lotto.domain;
+
+import static java.util.Arrays.stream;
+
+import java.util.List;
+import lotto.constant.Rank;
+
+public class ProfitRateCalculator {
+    public ProfitRateCalculator() {
+    }
+
+    public double calculateProfitRate(List<Rank> ranks, Money money) {
+        int totalGain = ranks.stream()
+                .map(Rank::getPrize)
+                .reduce(0, Integer::sum);
```

Comment:
`IntStream`을 이용하면 더 가독성이 좋은 코드를 작성할 수 있어요!!
```suggestion
                .mapToInt(Rank::getPrize)
                .sum();
```